### PR TITLE
add create/delete to specs and template. add time window delete to ratings

### LINKED ISSUE #275: Add post and delete to rating template endpoint
Used by CWMS clients

### LINKED ISSUE #276: Add post and delete to rating spec endpoint
Used by CWMS clients

### LINKED ISSUE #277: Delete rating endpoint needs a time window
Need to be able to delete a single rating effective date

### PR DESCRIPTION
**Pull request by adamkorynta**
Fixes: #275 
Fixes: #276 
Fixes: #277

### CODE REVIEW

**Comment by adamkorynta** [comment]
Need to add tests

File: cwms_radar_api/src/main/java/cwms/radar/api/RatingController.java [hunk 1]
```diff
@@ -1,22 +1,50 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
 package cwms.radar.api;
 
 import static com.codahale.metrics.MetricRegistry.name;
 import static cwms.radar.api.Controllers.AT;
 import static cwms.radar.api.Controllers.BEGIN;
+import static cwms.radar.api.Controllers.DATE_FORMAT;
```
Comment (MikeNeilson) [inline cwms_radar_api/src/main/java/cwms/radar/api/RatingController.java:30]: Arguably `import ...*` here would be okay.

**Review by MikeNeilson — COMMENTED** [review]
Looks good so far.

**Review by rma-rripken — DISMISSED** [review]

File: cwms_radar_api/src/main/java/cwms/radar/api/RatingController.java (hunk 1 above)
Comment (adamkorynta) [reply cwms_radar_api/src/main/java/cwms/radar/api/RatingController.java:30]: Yeah I had my IDE set to 999 for all * imports. Updated to allow constants to use *

**Review by rma-rripken — APPROVED** [review]

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**